lib/Net/Google/Drive/Simple.pm

```diff
@@ -5,22 +5,8 @@ package Net::Google::Drive::Simple;
 use strict;
 use warnings;
 
-use LWP::UserAgent ();
-use HTTP::Request  ();
-
-use File::MMagic ();
-use IO::File     ();
-
-use OAuth::Cmdline::CustomFile  ();
-use OAuth::Cmdline::GoogleDrive ();
-
-use Net::Google::Drive::Simple::Item ();
-
-use File::Basename;
-
-use YAML qw( LoadFile DumpFile );
-use JSON qw( from_json to_json );
-use Log::Log4perl qw(:easy);
+use Net::Google::Drive::Simple::V2;
+use Net::Google::Drive::Simple::V3;
```

Comment:
I don't think this is a good idea at the moment. V3 loads the same as V2, so at the moment, it's best to assure both load correctly instead of postponing this to instantiation time and then it's too late.